sentinel-adapter/sentinel-apache-dubbo-adapter/src/main/java/com/alibaba/csp/sentinel/adapter/dubbo/SentinelDubboProviderFilter.java

```diff
@@ -61,15 +61,13 @@ public Result invoke(Invoker<?> invoker, Invocation invocation) throws RpcExcept
         Entry methodEntry = null;
         try {
             String methodResourceName = DubboUtils.getResourceName(invoker, invocation, DubboConfig.getDubboProviderPrefix());
-            String interfaceResourceName = DubboConfig.getDubboInterfaceGroupAndVersionEnabled() ? invoker.getUrl().getColonSeparatedKey()
-                    : invoker.getInterface().getName();
+            String interfaceResourceName = DubboUtils.getInterfaceName(invoker);
             // Only need to create entrance context at provider side, as context will take effect
             // at entrance of invocation chain only (for inbound traffic).
             ContextUtil.enter(methodResourceName, application);
             interfaceEntry = SphU.entry(interfaceResourceName, ResourceTypeConstants.COMMON_RPC, EntryType.IN);
-            rpcContext.set(DubboUtils.DUBBO_INTERFACE_ENTRY_KEY, interfaceEntry);
             methodEntry = SphU.entry(methodResourceName, ResourceTypeConstants.COMMON_RPC, EntryType.IN, invocation.getArguments());
-            rpcContext.set(DubboUtils.DUBBO_METHOD_ENTRY_KEY, methodEntry);
+            rpcContext.set(methodResourceName, new Entry[]{interfaceEntry, methodEntry});
             return invoker.invoke(invocation);
```

Comment:
It's good adding a `DubboUtils#getInterfaceName` for resue.
Now the interfaceResource and methodResource key are merged into one key, but the key name is methodResourceName, is it better using a fixed key? @sczyh30 How about your opinion?

Reply:
Sorry for wrong idea, fixed key is incorrect here, since a method may have more than one consumer call with sync or async. I'm agree with adding the Sentinel prefix for identification.